cirq-core/cirq/experiments/qubit_characterizations.py

```diff
@@ -92,13 +93,32 @@ def plot(self, ax: Optional[plt.Axes] = None, **plot_kwargs: Any) -> plt.Axes:
             fig, ax = plt.subplots(1, 1, figsize=(8, 8))  # pragma: no cover
             ax = cast(plt.Axes, ax)  # pragma: no cover
         ax.set_ylim((0.0, 1.0))  # pragma: no cover
-        ax.plot(self._num_cfds_seq, self._gnd_state_probs, 'ro-', **plot_kwargs)
+        ax.plot(self._num_cfds_seq, self._gnd_state_probs, 'ro', **plot_kwargs)
+        x = np.linspace(self._num_cfds_seq[0], self._num_cfds_seq[-1], 100)
+        fit = self._fit_exponential()
+        ax.plot(x, fit[0][0] * fit[0][2] ** x + fit[0][1], '--k')
```

Comment:
nit: maybe adding a `label=''` would be good for understanding with this curve representing during visualization.